.github/workflows/ci.yml

```diff
@@ -7,6 +7,9 @@ on:
       - main
   workflow_dispatch:
     inputs:
+      amdgpu_families:
+        type: string
+        default: gfx94X-dcgpu
```

Comment:
Tagging https://github.com/ROCm/TheRock/issues/221 here since this is making progress towards that :D